scripts/netclient.sh

```diff
@@ -15,21 +27,26 @@ trap 'cleanup' SIGTERM
 
 # install netclient
 echo "[netclient] starting netclient daemon"
-/root/netclient install
+$_netclient install
 wait $!
 
 # check if needs to use the gui server
 if [ "${GUI_SERVER_ENABLED}" == "true" ]; then
     echo "[netclient] enabling gui server"
-    netclient guiServer enable
+    $_netclient guiServer enable
 else
     echo "[netclient] disabling gui server"
-    netclient guiServer disable
+    $_netclient guiServer disable
 fi
 
 # join network based on env vars
 echo "[netclient] joining network"
 
+SERVER_CMD=""
+if [ "$SERVER" != "" ]; then
+    SERVER_CMD="-s $SERVER"
```

Comment:
this flag is used only in case of SSO login, you join token/ enrolment key already has server information